app/src/main/java/org/oppia/app/recyclerview/DragAndDropItemFacilitator.kt

```diff
@@ -34,6 +38,11 @@ class DragAndDropItemFacilitator(
 
   override fun clearView(recyclerView: RecyclerView, viewHolder: RecyclerView.ViewHolder) {
     viewHolder.itemView.alpha = ALPHA_FULL
+    onItemDragListener.onItemDragged(
```

Comment:
Suggest using a separate callback for when the drag is completed rather than using a sentinel value. It tends to result in cleaner code when you avoid sentinel values.

Reply:
We can use a seperate function but in that case we wouldn't be able to lambda function inside data binding infact we would be needing to bind the interface with the data binding

Reply:
Perhaps use two interfaces, then? It's more cumbersome, but it keeps the API & usage clean.

Reply:
done